pkg/obfuscate/sql.go

```diff
@@ -16,9 +16,114 @@ import (
 
 var questionMark = []byte("?")
 
+// metadataFinderFilter is a filter which attempts to collect metadata from a query, such as comments and tables.
+// It is meant to run before all the other filters.
+type metadataFinderFilter struct {
+	collectTableNames bool
+	collectCommands   bool
+	collectComments   bool
+	replaceDigits     bool
+
+	// size holds the byte size of the metadata collected by the filter.
```

Comment:
This is wrong (documentation wise) because it excludes the tablesCSV size. IMHO we should include it, just for the sake integrity.

Reply:
https://github.com/DataDog/datadog-agent/pull/10042/commits/6be76eac6b841793ddd5162f904ee8b1557c4f2c